numpy/fft/__init__.py

```diff
@@ -118,8 +118,16 @@
 It differs from the forward transform by the sign of the exponential
 argument and the default normalization by :math:`1/n`.
 
+Type Promotion
+--------------
+
+`numpy.fft` is inconsistent with other NumPy functions as it promotes
+`float32` and `complex64` arrays to `float64` and `complex128` arrays.
+For an FFT implementation that does not promote input arrays, see `scipy.fftpack`.
+
```

Comment:
I think 
```
"`numpy.fft` promotes ``float32`` and ``complex64`` arrays to ``float64`` and
``complex128`` arrays respectively."
```

 reads better. Note also the double backticks needed for the dtypes.